diff --git a/dist/chunks/run.BTlFXlnv.js b/dist/chunks/run.BTlFXlnv.js
index ef96034d2cb39ec3c1cf534e6ad27f11621ab690..59a4936ab05569e6bdd7887fc8e397661f691bf9 100644
--- a/dist/chunks/run.BTlFXlnv.js
+++ b/dist/chunks/run.BTlFXlnv.js
@@ -2998,7 +2998,7 @@ function formatTitle(template, items, idx) {
 	const inspectOptions = { truncate: runner.config.taskTitleValueFormatTruncate };
 	const isObjectItem = isObject(items[0]);
 	function formatAttribute(s) {
-		return s.replace(/\$([$\p{ID_Continue}.]+)/gu, (_, key) => {
+		return s.replace(/\$([$\w.]+)/g, (_, key) => {
 			const isArrayKey = /^\d+$/.test(key);
 			if (!isObjectItem && !isArrayKey) return `$${key}`;
 			const arrayElement = isArrayKey ? objectAttr(items, key) : void 0;
